config/config.go

```diff
@@ -82,13 +83,7 @@ type Extensions struct {
 // executing signer actions specific for TBTC application.
 type TBTC struct {
 	TBTCSystem string
-	BTCRefunds BTCRefunds
-}
-
-// BTCRefunds stores configuration related to recovering BTC from a closed keep.
-type BTCRefunds struct {
-	BeneficiaryAddress string
-	MaxFeePerVByte     int32
+	BTCRefunds bitcoin.Config
```

Comment:
Just flagging that the entire `TBTC` struct with `BTCRefunds` inside should be 🔥 after the `master` reverse-merge, and `bitcoin.Config` should be a standalone part of the config with its own `[bitcoin]` namespace in the TOML file.